.github/workflows/build.yml

```diff
@@ -4,60 +4,352 @@ name: Build APBS
 on:
   pull_request:
   push:
-    branches: [master]
+    branches: [main]
+    tags: ['*']
+
+env:
+  WIN_DEPENDENCIES_RELEASE: "v0.4.0"
+  WIN_TOOLCHAIN_FILE: '/c/vcpkg/scripts/buildsystems/vcpkg.cmake'
+  BLA_VENDOR: OpenBLAS
+  BUILD_DOC: ON
+  BUILD_SHARED_LIBS: OFF
+  BUILD_TOOLS: ON
+  RELEASE_TYPE: Release
+  ENABLE_PYGBE: ON
+  ENABLE_BEM: ON
+  ENABLE_GEOFLOW: ON
+  ENABLE_FETK: ON
+  FETK_VERSION: "1.8.1"
+  ENABLE_iAPBS: ON
+  ENABLE_OPENMP: ON
+  ENABLE_PBAM: ON
+  ENABLE_PBSAM: ON
+  ENABLE_PYTHON: OFF
+  ENABLE_TESTS: ON
+  GIT_SUBMODULE: OFF
```

Comment:
@nsoblath you can probably remove all of the GIT_SUBMODULE code since we don't use it anymore.